### PR TITLE
Updated the md/Rmd files from google docs again.

### PR DESCRIPTION
**Pull request by grabear**

### CODE REVIEW

File: proposal.Rmd [hunk 1]
```diff
@@ -1,110 +1,110 @@
...
...ent](https://github.com/npm/cli) for the [npm-registry](https://docs.npmjs.com/misc/registry). Ruby has the [gem-client](https://github.com/rubygems/rubygems) for [RubyGems.org](https://rubygems.org/). R natively provides the ability to use *R CMD INSTALL* for tarballs and *install.packages()* for CRAN packages, but not a native option for installing specific versions of packages or installing from GitHub.In addition to package management, researchers often need to locally install software. HPC systems, for instance, may lack required software, have fewer choices for different versions of software, or have the software configured ineffectively for the user's needs. This is especially true for programming languages that frequently receive updates to their source code. To alleviate this, Python and Ruby have third-party interfaces ([pyenv](https://github.com/pyenv/pyenv), [rbenv](https://github.com/rbenv/rbenv), and [ruby-build](https://github.com/jdblischak/workflowr)) that manage the installation and management of various versions of the programming language. While [conda](https://conda.io/docs/) can manage to install R, it cannot manage multiple installed versions of the language.
```
Comment (santina) [inline proposal.Rmd]:
```suggestion
\indent\indent Most programming languages commonly used by the data science community have a command-line package manager, which offers access to a standard or custom package repository. Python, for example, has [pip](https://pypi.org/project/pip/) for [PyPi](https://pypi.org/) and [conda](https://conda.io/docs/) for [Anaconda Cloud](https://anaconda.org/). JavaScript has the [npm-client](https://github.com/npm/cli) for the [npm-registry](https://docs.npmjs.com/misc/registry). Ruby has the [gem-client](https://github.com/rubygems/rubygems) for [RubyGems.org](https://rubygems.org/). R natively provides the ability to use *R CMD INSTALL* for tarballs and *install.packages()* for CRAN packages, but not a native option for installing specific versions of packages or installing from GitHub. In addition to package management, researchers often need to locally install software. HPC systems, for instance, may lack required software, have fewer choices for different versions of software, or have the software configured ineffectively for the user's needs. This is especially true for programming languages that frequently receive updates to their source code. To alleviate this, Python and Ruby have third-party interfaces ([pyenv](https://github.com/pyenv/pyenv), [rbenv](https://github.com/rbenv/rbenv), and [ruby-build](https://github.com/jdblischak/workflowr)) that manage the installation and management of various versions of the programming language. While [conda](https://conda.io/docs/) can manage to install R, it cannot manage multiple installed versions of the language.
```

File: proposal.Rmd [hunk 2]
```diff
@@ -1,110 +1,110 @@
...
+\indent\indent The main goals for the beRi CLI include seamlessly integrating renv, rinse, and rut by establishing a unified directory structure and creating a portable project workflow. By creating a user level configuration directory,, we hope to enhance our ability to configure R for beRi projects. The \~/.beri folder, for instance, will contain any renv environments, rinse R installations, and rut package installations associated with beRi processes by default. Our configuration and management process for beRi projects will involve the use of YAML files for each standalone package (*renv.yaml, rinse.yaml, and rut.yaml*). A top-level YAML file will also be used with beRi (*beRi.yaml*) as a master configuration file for making our projects portable.
```
Comment (santina) [inline proposal.Rmd]:
```suggestion
\indent\indent The main goals for the beRi CLI include seamlessly integrating renv, rinse, and rut by establishing a unified directory structure and creating a portable project workflow. By creating a user level configuration directory, we hope to enhance our ability to configure R for beRi projects. The \~/.beri folder, for instance, will contain any renv environments, rinse R installations, and rut package installations associated with beRi processes by default. Our configuration and management process for beRi projects will involve the use of YAML files for each standalone package (*renv.yaml, rinse.yaml, and rut.yaml*). A top-level YAML file will also be used with beRi (*beRi.yaml*) as a master configuration file for making our projects portable.
```

**Review by santina — CHANGES_REQUESTED** [review]
You are missing a space in the first one and have two commas in the second one.

File: proposal.Rmd (hunk 2 above)
Comment (santina) [reply proposal.Rmd]: Highlighting didn't work ... You have 
>"By creating a user level configuration directory, , we "

File: proposal.Rmd (hunk 1 above)
Comment (santina) [reply proposal.Rmd]: You have 
> r installing specific versions of packages or installing from GitHub.In addition ...

File: proposal.Rmd (hunk 2 above)
Comment (grabear) [reply proposal.Rmd]: Good eye!  @santina

File: proposal.Rmd [hunk 3]
```diff
@@ -1,110 +1,110 @@
...
-  html_document: default
   pdf_document: default
+  html_document: default
+  word_document: default
 ---
 
-**[Robert Gilmore](https://github.com/grabear)^1^,
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**[Robert Gilmore](https://github.com/grabear)^1^,
 [Shaurita Hutchins](https://github.com/sdhutchins)^1^,
 [Bruno Grande](https://github.com/edzer)^2^,
```
Comment (santina) [inline proposal.Rmd]:
```suggestion
[Bruno Grande](https://github.com/scientificbruno)^2^,
```

**Review by santina — CHANGES_REQUESTED** [review]

File: proposal.md [hunk 4]
```diff
@@ -1,152 +1,373 @@
-beRi Environments for R Installations
+BeRi Environments for R Installations
 ================
 October 28th, 2018
 
-**[Robert Gilmore](https://github.com/grabear)<sup>1</sup>, [Shaurita Hutchins](https://github.com/sdhutchins)<sup>1</sup>, [Bruno Grande](https://github.com/edzer)<sup>2</sup>, [Santina Lin](https://github.com/santina)<sup>3</sup>**
+                **[Robert
+Gilmore](https://github.com/grabear)<sup>1</sup>, [Shaurita
+Hutchins](https://github.com/sdhutchins)<sup>1</sup>, [Bruno
+Grande](https://github.com/edzer)<sup>2</sup>, [Santina
```
Comment (santina) [inline proposal.md]:
```suggestion
Grande](https://github.com/scientificbruno)<sup>2</sup>, [Santina
```

File: proposal.md [hunk 5]
```diff
@@ -1,152 +1,373 @@
...
+[pip](https://pypi.org/project/pip/) for [PyPi](https://pypi.org/) and
+[conda](https://conda.io/docs/) for [Anaconda
+Cloud](https://anaconda.org/). JavaScript has the
+[npm-client](https://github.com/npm/cli) for the
+[npm-registry](https://docs.npmjs.com/misc/registry). Ruby has the
+[gem-client](https://github.com/rubygems/rubygems) for
+[RubyGems.org](https://rubygems.org/). R natively provides the ability
+to use *R CMD INSTALL* for tarballs and *install.packages()* for CRAN
+packages, but not a native option for installing specific versions of
+packages or installing from GitHub.In addition to package management,
```
Comment (santina) [inline proposal.md]:
```suggestion
packages or installing from GitHub. In addition to package management,
```

File: proposal.md [hunk 6]
```diff
@@ -1,152 +1,373 @@
...
+[jetpack](https://github.com/ankane/jetpack), or utilizing
+[rpy2](https://rpy2.readthedocs.io/en/version_2.8.x/) for getting
+interpreter specific information.
+
+## beRi (Integrated CLI)
+
+The main goals for the beRi CLI include seamlessly integrating renv,
+rinse, and rut by establishing a unified directory structure and
+creating a portable project workflow. By creating a user level
+configuration directory,, we hope to enhance our ability to configure R
```
Comment (santina) [inline proposal.md]:
```suggestion
configuration directory, we hope to enhance our ability to configure R
```

**Review by santina — CHANGES_REQUESTED** [review]

**Review by BrunoGrandePhD — APPROVED** [review]

**Review by santina — APPROVED** [review]

**Pull request merged**